mw-webapp/src/dataAccessLogic/DTOToBusinessConverter/mentorCommentDTOToMentorCommentConverter.ts

```diff
@@ -0,0 +1,30 @@
+import {MentorComment} from "src/model/businessModel/MentorComment";
+import {MentorCommentDTO} from "src/model/firebaseCollection/MentorCommentDTO";
+import {UserDTO} from "src/model/firebaseCollection/UserDTO";
+
+
+/**
+ * Convert {@link MentorCommentDTO} to {@link MentorComment}
+ * @param {MentorCommentDTO} mentorCommentDTO {@link MentorCommentDTO}
+ * @param {UserDTO[]} usersDTO - {@link UserDTO}
+ * @returns {MentorComment} mentorComment {@link MentorComment}
+ */
+export const MentorCommentDTOToMentorCommentConverter = (
+  mentorCommentDTO: MentorCommentDTO, usersDTO: UserDTO[],
+): MentorComment => {
+  const matchingUser = usersDTO.find(user => user.uuid === mentorCommentDTO.mentorUuid);
```

Comment:
Is i tpossible to write this in one line?

Reply:
133 lenght, we have 130 max